1.0/nanoserver/sdk/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM microsoft/dotnet:1.0.4-sdk-nanoserver-10.0.14393.953
 # Install node
 ENV NODE_VERSION 6.10.2
 
-RUN Invoke-WebRequest https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
+RUN Invoke-WebRequest -UseBasicParsing https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
```

Comment:
It is my understanding that this is by design.  iwr is using IE to parse the DOM under the covers.  IE is not present on NanoServer. -UseBasicParsing tells PS to use its own parser.  Here is the error message -

```
iwr : The response content cannot be parsed because the Internet Explorer engine is not available, or Internet
Explorer's first-launch configuration is not complete. Specify the UseBasicParsing parameter and try again.
```

Perhaps it would be better to make this the default behavior on Nano but that would probably confusion as well having different behavior across OSes.

Yes this works on Windows 10 Pro.